src/custom-elements/src/components/link/link.tsx

```diff
@@ -0,0 +1,102 @@
+import { Component, Element, Prop, State, Watch, h } from '@stencil/core';
+
+@Component({
+  tag: 'chi-link',
+  styleUrl: 'link.scss',
+  scoped: true
+})
+export class Link {
+  @State() slotLinkContent = true;
+
+  @Element() el: HTMLElement;
+
+  /**
+   *  to set a links destination url.
+   */
+  @Prop({ reflect: true }) href: string = '#';
+
+  /**
+   *  to set the language of a links destination url.
+   */
+  @Prop({ reflect: true }) hreflang: string;
+
+  /**
+   *  to provide alternative text in case of icon links.
+   */
+  @Prop({ reflect: true }) alternativeText: string;
+
+  /**
+   *  to render a link as a prominent call to action.
+   */
+  @Prop({ reflect: true }) cta = false;
+
+  /**
+   *  to disable a link.
+   */
+  @Prop({ reflect: true }) disabled = false;
+
+  /**
+   *  to prompt users to save a page or document instead of navigating to it.
+   */
+  @Prop({ reflect: true }) download: string;
+
+  /**
+   *  to set the relationship between the current page and the linked page.
+   */
+  @Prop({ reflect: true }) rel: string;
+
+  /**
+   *  to set size of a link { xs, sm, md, lg, xl }.
+   */
+  @Prop({ reflect: true }) size: string;
```

Comment:
Do we need the 'md' size, since it's the same as not setting a size?
Also, maybe it's a good idea to create a sizes' set type and defaulting this to 'md' in case we want to keep that value. Thoughts?

Reply:
@SergioQCL Links behave similar to text in that they inherit the font-size of the parent by default. This would allow a scenario where the default size and md size are not the same. For this reason I would suggest not defaulting to md.